ngraph/python/tests/test_onnx/test_backend.py

```diff
@@ -173,24 +169,17 @@ def expect_fail(test_case_path, xfail):  # type: (str) -> None
      "OnnxBackendNodeModelTest.test_nonmaxsuppression_two_classes_cpu"),
     (xfail_issue_39662,
      "OnnxBackendNodeModelTest.test_nonmaxsuppression_two_classes_cpu",
-     "OnnxBackendNodeModelTest.test_slice_default_axes_cpu",
      "OnnxBackendNodeModelTest.test_scatter_elements_with_negative_indices_cpu",
-     "OnnxBackendNodeModelTest.test_constant_pad_cpu",
-     "OnnxBackendNodeModelTest.test_edge_pad_cpu",
-     "OnnxBackendNodeModelTest.test_reflect_pad_cpu",
      "OnnxBackendNodeModelTest.test_top_k_cpu",
      "OnnxBackendNodeModelTest.test_top_k_negative_axis_cpu",
      "OnnxBackendNodeModelTest.test_top_k_smallest_cpu",
```

Comment:
Please double check issue label for TopK

Reply:
Done